mini_pose:
    - { 
        pose_name: "initial_pose",
        head_pan_joint: 0.0,
        head_tilt_joint: 0.0,     
        body_roll_joint: 0.0,                             
        right_shoulder_roll_joint: 0.0,
        right_shoulder_flex_joint: -0.82,
        right_elbow_roll_joint: 0.0,
        right_hand_motor_joint: 0.0,
        left_shoulder_roll_joint: 0.0,
        left_shoulder_flex_joint: -0.82,
        left_elbow_roll_joint: 0.0,
        left_hand_motor_joint : 0.0
    }

    - { 
        pose_name: "attention_pose",
        head_pan_joint: 0.0,
        head_tilt_joint: 0.0,       
        body_roll_joint: 0.0,                           
        right_shoulder_roll_joint: 0.0,
        right_shoulder_flex_joint: -1.30,
        right_elbow_roll_joint: 0.0,
        right_hand_motor_joint: 0.0,
        left_shoulder_roll_joint: 0.0,
        left_shoulder_flex_joint: -1.30,
        left_elbow_roll_joint: 0.0,
        left_hand_motor_joint : 0.0
    }

    - { 
        pose_name: "receive_pose", 
        head_pan_joint: -0.2,
        head_tilt_joint: 0.0,     
        body_roll_joint: 0.0,                            
        right_shoulder_roll_joint: 0.0,
        right_shoulder_flex_joint: -1.30,
        right_elbow_roll_joint: 1.57,
        right_hand_motor_joint: 0.0,
        left_shoulder_roll_joint: 0.0,
        left_shoulder_flex_joint: -1.30,
        left_elbow_roll_joint: 0.0,
        left_hand_motor_joint : 0.0
    }